integration/operator_test.go

```diff
@@ -277,7 +277,7 @@ func operatorSetup() error {
 		return microerror.Maskf(err, "installing aws-resource-lab chart")
 	}
 
-	logEntry := "creating AWS cloudformation stack: created"
+	logEntry := "created the guest cluster cloud formation stack"
```

Comment:
I really don't like this but right now I just comply. Would be cool to have something better for the future. This is just error prone. It is so easy to fall for that.